### DISCUSSION TITLE
Android 每次加载同一个pag文件缓存数据有区别吗

### DISCUSSION THREAD

**Discussion by thinylim** [Tencent/libpag#1615] [Q&A]
目前发现对同一个pag文件（1000*800），使用PAGImageView播放动效，会偶现启动时，有个闪屏现象，出现后，每次播放这个动效基本都复现，然后清了应用缓存数据后就不复现了
主要有几个疑问，希望能解答：
1、是不是加载这个pag数据出错导致的
2、每次加载文件数据逻辑是否都是一样的，会与系统环境有关系吗
3、为什么清了缓存数据后就正常了呢

**Comment by kevingpqi123** [comment]
这类问题建议基于 pag 的官方 demo 构造出复现场景，这样分析起来是最快的

**Comment by kevingpqi123** [comment]
一周未回复，暂时关闭
